fe/fe-core/src/main/java/com/starrocks/catalog/WorkGroupClassifier.java

```diff
@@ -109,7 +109,7 @@ public boolean isVisible(String user, String role, String sourceIp) {
         if (this.role != null && !this.role.equals(role)) {
```

Comment:
needn't ignore case here？

Reply:
if role condition is present, then it is must be matched strictly.